theano/tensor/tests/test_opt.py

```diff
@@ -5878,19 +5878,28 @@ def test_local_useless_split():
 
 
 def test_local_flatten_lift():
+    """
+    .. note:: The Flatten(Op) is deprecated, and this method
+        should be removed with Flatten.
```

Comment:
I don't think this is the right place for that note, since the test has been updated to use `tensor.flatten` rather than the `Flatten` Op. Is it supposed to go somewhere else?